routes/api_v1_0.py

```diff
@@ -325,20 +325,54 @@ def get_data_v1_0():
     except ValidationError as e:
         abort(400, str(e))
 
+    cached_file_name = os.path.join(
+        current_app.config['CACHE_DIR'],
+        "data",
+        f"get_data_{result['dataset']}_{result['variable']}_{result['depth']}_{result['time']}_{result['geometry_type']}.geojson"
+        )
+    
+    if os.path.isfile(cached_file_name):
+        print(f"Using cached {cached_file_name}")
+        with open(cached_file_name, 'r') as f:
+            send_file(f, 'application/json')
+
     config = DatasetConfig(result['dataset'])
     
     with open_dataset(config, variable=result['variable'], timestamp=result['time']) as ds:
-        return jsonify(
-            geojson.dumps(
-                data_array_to_geojson(
-                    ds.nc_data.get_dataset_variable(result['variable'])[result['time'], result['depth'], :, :],
-                    config.lat_var_key,
-                    config.lon_var_key
-                ),
-                allow_nan=True
+
+        lat_var, lon_var = ds.nc_data.latlon_variables
+
+        lat_slice = slice(0, lat_var.size, 10)
+        lon_slice = slice(0, lon_var.size, 8)
```

Comment:
Grab every 10th latitude, and every 8th longitude. Increasing these values reduces the number of data points sent back to the browser. Decreasing these values will increase the number of data points returned....by a lot. Use with care.